### PR TITLE
chore(i18n): adapted lowercase

### PR DESCRIPTION
**Pull request by rakekniven**
Reported at Transifex

### CODE REVIEW

File: apps/settings/src/components/AdminAI.vue [hunk 1]
```diff
@@ -93,7 +93,7 @@
 			<template v-if="tpTaskTypes.length === 0">
 				<NcNoteCard type="info">
 					<!-- TRANSLATORS Text processing is the name of a Nextcloud-internal API -->
-					{{ t('settings', 'None of your currently installed apps provide custom Text processing functionality.') }}
+					{{ t('settings', 'None of your currently installed apps provide custom text processing functionality.') }}
```
Comment (marcelklehr) [inline apps/settings/src/components/AdminAI.vue:96]: Isn't the convention to use upper case if it's a name?

Comment (solracsf) [inline apps/settings/src/components/AdminAI.vue:96]:
```suggestion
					{{ t('settings', 'None of your currently installed apps provide custom Text API processing functionality.') }}
```
Maybe?

Comment (marcelklehr) [reply apps/settings/src/components/AdminAI.vue:96]: The API is called "Text Processing API"

Comment (marcelklehr) [reply apps/settings/src/components/AdminAI.vue:96]: Ah, that's better, yeah :)

**Review by solracsf — APPROVED** [review]

**Review by marcelklehr — APPROVED** [review]

**Pull request merged**